test/lib/ufe/testGroupCmd.py

```diff
@@ -310,6 +362,166 @@ def testGroupRestirction(self):
             stage.GetPrimAtPath("/Ball_set/Props/Ball_6"),
             stage.GetPrimAtPath("/Sphere1")])
 
+    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '3005', 'testGroupAbsolute is only available in UFE preview version 0.3.5 and greater')
+    def testGroupAbsolute(self):
+        '''Verify -absolute flag.'''
+
+        cmds.file(new=True, force=True)
+
+         # create a stage
+        (stage, proxyShapePathStr, proxyShapeItem, contextOp) = createStage();
+
+        # create a sphere generator
+        sphereGen = SphereGenerator(1, contextOp, proxyShapePathStr)
+
+        spherePath = sphereGen.createSphere()
+        spherePrim = mayaUsd.ufe.ufePathToPrim(ufe.PathString.string(spherePath))
+
+        # no TRS attributes
+        self.assertFalse(spherePrim.HasAttribute('xformOp:translate'))
+        self.assertFalse(spherePrim.HasAttribute('xformOp:rotateXYZ'))
+        self.assertFalse(spherePrim.HasAttribute('xformOp:scale'))
+
+        # create a group with absolute flag set to True
+        cmds.group(ufe.PathString.string(spherePath), absolute=True)
+
+        # verify that groupItem has 1 child
+        groupItem = ufe.GlobalSelection.get().front()
+        groupHierarchy = ufe.Hierarchy.hierarchy(groupItem)
+        self.assertEqual(len(groupHierarchy.children()), 1)
+
+        # verify XformOpOrderAttr exist after grouping
+        newspherePrim = stage.GetPrimAtPath("/group1/Sphere1")
+        sphereXformable = UsdGeom.Xformable(newspherePrim)
+
+        self.assertEqual(
+            sphereXformable.GetXformOpOrderAttr().Get(), Vt.TokenArray((
+                "xformOp:translate", "xformOp:rotateXYZ", "xformOp:scale")))
+
+
+    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '3005', 'testGroupRelative is only available in UFE preview version 0.3.5 and greater')
+    def testGroupRelative(self):
+        '''Verify -relative flag.'''
+        cmds.file(new=True, force=True)
+
+         # create a stage
+        (stage, proxyShapePathStr, proxyShapeItem, contextOp) = createStage();
+
+        # create a sphere generator
+        sphereGen = SphereGenerator(1, contextOp, proxyShapePathStr)
+
+        spherePath = sphereGen.createSphere()
+        spherePrim = mayaUsd.ufe.ufePathToPrim(ufe.PathString.string(spherePath))
+
+        # no TRS attributes
+        self.assertFalse(spherePrim.HasAttribute('xformOp:translate'))
+        self.assertFalse(spherePrim.HasAttribute('xformOp:rotateXYZ'))
+        self.assertFalse(spherePrim.HasAttribute('xformOp:scale'))
+
+        # create a group with absolute flag set to True
```

Comment:
Copy paste damage in comment.

Reply:
7705f64f4beb807d3dfa269c41766520ae8521a0